src/Microsoft.OData.Client/Materialization/ODataEntriesEntityMaterializer.cs

```diff
@@ -58,6 +58,14 @@ internal override ODataResourceSet CurrentFeed
             get { return null; }
         }
 
+        /// <summary>
+        /// OData delta resource set being materialized; possibly null.
+        /// </summary>
+        internal override ODataDeltaResourceSet CurrentDeltaResourceSetFeed

```

Comment:
```suggestion
        internal override ODataDeltaResourceSet CurrentDeltaFeed
```

Reply:
It's not  used anywhere. getting rid of it